compose-post.sh

```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/env bash
 
-set -e
+set -xe
+
+# See machineid-compat in host.yaml.
+# Since that makes presets run on boot, we need to have our defaults in /usr
+ln -sfr /usr/lib/systemd/system/{multi-user,default}.target
+# This should be in our -release package, but let's do this here for now
```

Comment:
how do we track removing this workaround and putting it in the proper place?

Reply:
I'd say make an issue and tag it into Jira so it's tech debt that doesn't get lost.

Reply:
https://github.com/openshift/redhat-release-coreos/pull/3